src/backend/quarkus-management-api/src/main/java/com/nttdata/quarkus/management/api/model/api/Contact.java

```diff
@@ -0,0 +1,207 @@
+package com.nttdata.quarkus.management.api.model.api;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+import javax.validation.Valid;
+import java.math.BigDecimal;
+import java.util.Objects;
+
+/**
+ * Contact
+ */
+@ApiModel(description = "Contact")
+@javax.annotation.Generated(value = "org.openapitools.codegen.languages.SpringCodegen", date = "2020-10-20T07:29:34.739-07:00[America/Los_Angeles]")
+
+public class Contact {
+  @JsonProperty("contactId")
+  private BigDecimal contactId;
+
+  @JsonProperty("firstName")
+  private String firstName;
+
+  @JsonProperty("lastName")
+  private String lastName;
+
+  @JsonProperty("email")
+  private String email;
+
+  @JsonProperty("phoneNumber")
+  private String phoneNumber;
+
+  @JsonProperty("customerId")
+  private BigDecimal customerId;
+
+  public Contact contactId(BigDecimal contactId) {
+    this.contactId = contactId;
+    return this;
+  }
+
+  /**
+   * Get contactId
+   * @return contactId
+  */
+  @ApiModelProperty(value = "")
+
+  @Valid
```

Comment:
question for personal knowledge:
what are these two lines/annotations doing?

Reply:
These are auto gen'd @Valid ensure valid value the other not sure

Reply:
The @ApiModelProperty annotation allows us to control Swagger-specific definitions such as description (value), name, data type, example values, and allowed values for the model properties